Automattic/wp-calypso
# Reader: Data-Layer for recommended sites

### Summary
The manage refresh (Search on Manage) requires us to tap into our new recommended sites api.
This PR seeks to add in the necessary data-layer/actions/reducers etc. for requesting recommended sites from the api.  


State shape:
```
{ 
  seed: [ { feedId, url, title }, site2, site3 ]
}
```


### To test:
1. go to http://calypso.localhost:3000/
2. run 
```
dispatch({ 
  type: 'READER_RECOMMENDED_SITES_REQUEST',
  payload: {
   seed: 0,
  }
 })
```
3. confirm the network request went out as expected.  Then also check that it shows up properly in the state tree.
